src/desktop/apps/artist/client.tsx

```diff
@@ -15,13 +16,15 @@ buildClientApp({
   } as any,
 })
   .then(({ ClientApp }) => {
-    ReactDOM.hydrate(
-      <ClientApp />,
-      document.getElementById("react-root"),
-      () => {
-        setupArtistSignUpModal()
-      }
-    )
+    loadableReady(() => {
```

Comment:
@zephraph - this was indeed needed to prevent flash between ssr and clientside hydration